faker/providers/python/__init__.py

```diff
@@ -48,7 +48,7 @@ def pystr(self, min_chars=None, max_chars=20):
             )
 
     def pystr_format(self, string_format='?#-###{{random_int}}{{random_letter}}', letters=string.ascii_letters):
-        return self.bothify(self.generator.parse(string_format), letters=letters)
+        return self.generator.parse(string_format)
```

Comment:
this way won't we loose the option of passing a custom alphabet with `letters=`? Do we have any provider using this feature?